package-lock.json

```diff
@@ -5,9 +5,9 @@
   "requires": true,
   "dependencies": {
     "@alfresco/adf-cli": {
-      "version": "3.6.0",
-      "resolved": "https://registry.npmjs.org/@alfresco/adf-cli/-/adf-cli-3.6.0.tgz",
-      "integrity": "sha512-HewDSVxz+BGqSTlooOj2jgnX5g5lNSXwsI8K+eqr55s12ku7AeX0JRFxrvVz20v/2efGMnwHHXGfF8tbdCChnA==",
+      "version": "3.7.0-beta6",
+      "resolved": "https://agora.dequecloud.com/artifactory/api/npm/dequelabs/@alfresco/adf-cli/-/adf-cli-3.7.0-beta6.tgz",
```

Comment:
I think this is not OK to be committed like this, using deque paths ....

Reply:
wow i'm not even sure how that happened! yeah no kidding!